services/harmony/test/ogc-api-edr/get-data-for-cube.ts

```diff
@@ -0,0 +1,1022 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import _ from 'lodash';
+import isUUID from '../../app/util/uuid';
+import { itRedirectsToJobStatusUrl } from '../helpers/jobs';
+import { hookPostEdrRequest, hookEdrRequest, edrRequest } from '../helpers/ogc-api-edr';
+import hookServersStartStop from '../helpers/servers';
+import StubService, { hookServices } from '../helpers/stub-service';
+import { ServiceConfig } from '../../app/models/services/base-service';
+import { hookRedirect } from '../helpers/hooks';
+import { stub } from 'sinon';
+import env from '../../app/util/env';
+import { hookDatabaseFailure } from '../helpers/db';
+
+describe('OGC API EDR - getEdrCube', function () {
```

Comment:
I think when we add other spatial queries besides `cube` that we should try to avoid just duplicating all these tests for the new query type. The handling for variables, crs, etc. is probably going to be the same code for each spatial type, so maybe we can separate them out and just test them in one place, then have separate tests that only test the spatial stuff. Don't need to to this now, but we should discuss it at some point.

Reply:
Let's refactor this when we add more routes, not now.